### PR TITLE
[python3-babel] Package babel. JB#58729

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: rpm/python3-babel.spec [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Requires: %{name} = %{version}-%{release}
+
+%description -n babel
+Tools for internationalizing Python applications
+
+%prep
+%autosetup -n %{name}-%{version}/%{name}
+
+%build
+cp ../rpm/cldr-common-41.0.zip cldr/
```
Comment (martyone) [inline rpm/python3-babel.spec]:
```suggestion
cp %{SOURCE1} cldr/
```

File: rpm/python3-babel.spec [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+%python3 ./setup.py import_cldr
+%py3_build
+
+%install
+%py3_install
+
+%files
+%defattr (-,root,root,-)
+%license LICENSE cldr/cldr-common-41.0/LICENSE.txt
```
Comment (martyone) [inline rpm/python3-babel.spec]: Maybe use wildcard for easier maintenance?

**Review by martyone — APPROVED** [review]
Two suggestions, then LGTM

File: rpm/python3-babel.spec (hunk 2 above)
Comment (vigejolla) [reply rpm/python3-babel.spec]: Which part would you replace with wildcard? The cldr version number?

Comment (martyone) [reply rpm/python3-babel.spec]: yup

**Pull request merged**